README.md

```diff
@@ -1 +1,181 @@
+- **Demonstrate ability to create UIs adaptable to different screen sizes using media queries.**
+- **Use GitHub Pages to deploy web pages.**
+- **Apply JavaScript best practices and language style guides in code.**
+- **Use JavaScript to manipulate DOM elements.**
+- **Use JavaScript events.**
+- **Use objects to store and access data.**
+- **Communicate technical concepts to other technical people.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://dieum512.github.io/HTML-basic-JavaScript-Capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom Video <a name="loom-video"></a>
+
+- [Loom Video Link](https://www.loom.com/share/126496b7c98343cd9ec44fe920c07f48)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A code compiler like visual studio code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+use git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+npm
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start or live server
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@dieum512](https://github.com/dieum512/)
+- Twitter: [@dieumakonkw](https://twitter.com/dieumakonkw)
+- LinkedIn: [Dieum Akonkwa](https://www.linkedin.com/in/dieum-akonkwa-7981b924b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Finish the mobile version]**
+- [ ] **[Make some changes]**
+- [ ] **[link the social icons]**
+- [ ] **[Finish the desktop version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this opportunity of practicing
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] I think it can be better if you add the MIT License to your README file so then it can look more professional  😎, the link that you provided is broken 🙃

Reply:
Fixed.